helm/hpcc/values.yaml

```diff
@@ -53,6 +53,11 @@ global:
   # Set postJobCommandViaSidecar to true, if the command needs to run with privilege, this will enable the command
   # to run as root in a sidecar in same process space as other containers, allowing it to for example send signals
   # to processes in sidecars
+  # For Istio or similar:
```

Comment:
You seem to have lost the misc: part

Reply:
This PR is no more valid since "metrics" was removed by the other PR. I will close this PR and JIRA